repository/t.json

```diff
@@ -3152,7 +3152,11 @@
 					"tags": "st2-"
 				},
 				{
-					"sublime_text": ">=3148",
+					"sublime_text": "3147 - 4107",
```

Comment:
I believe this range syntax was not inclusive (correct me if I'm wrong) so it might appear like it overlaps with st2 and st4 versions.